docs/components/movement-sensor/merged.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Configure an merged movement sensor"
+linkTitle: "merged"
+weight: 40
+type: "docs"
+description: "Configure an merged movement sensor."
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand
+---
+
+The `merged` movement sensor model supports a movement sensor that allows you to get multiple types of [`Readings`](/components/movement-sensor/#getreadings), including angular and linear velocity, from *one* movement sensor {{< glossary_tooltip term_id="resource" text="resource" >}} on your robot.
+
+This allows you to aggregate the API methods supported by multiple sensors into a singular sensor client, effectively merging the models of the individual resources.
+
+In other words, the models of the `merged` sensors are combined, making the movement sensor API methods each aggregated model supports available on the `merged` model.
```

Comment:
```suggestion
```
I don't think this helps, because this is already stated, plus it doesn't make it any more clear that they actually do still need to configure the other components first.

Reply:
I do think it helps bc of the model server distinction, but I agree this could be shortened

Reply:
I mean that I want people to be aware that the model is serving through API

Reply:
I think the first two sentences already cover it though